modules/nvidia-gpu-admin-dashboard-installing.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * monitoring/nvidia-gpu-admin-dashboard.adoc
+
+:_content-type: PROCEDURE
+[id="nvidia-gpu-admin-dashboard-installing_{context}"]
+= Installing the NVIDIA GPU administration dashboard
+
+Install the NVIDIA GPU plugin using Helm on the OpenShift Container 
+Platform (OCP) Console to add GPU capabilities.
+
+The OpenShift Console NVIDIA GPU plugin works as a remote bundle for the OCP console. To run the OpenShift Console NVIDIA GPU plugin
+an instance of the OCP console must be running.
+
+
+.Prerequisites
+
+* Red Hat OpenShift 4.11+
+* NVIDIA GPU operator
+* link:https://helm.sh/docs/intro/install/[Helm]
+
+
+.PROCEDURE
+
+Use the following procedure to deploy the OpenShift Console NVIDIA GPU plugin.
+
+. Add the Helm repository:
++
+[source,terminal]
+----
+$ helm repo add rh-ecosystem-edge https://rh-ecosystem-edge.github.io/console-plugin-nvidia-gpu
+----
++
+[source,terminal]
+----
+$ helm repo update
+----
+
+. Install the Helm chart in the default NVIDIA GPU operator namespace:
++
+[source,terminal]
+----
+$ helm install -n nvidia-gpu-operator console-plugin-nvidia-gpu rh-ecosystem-edge/console-plugin-nvidia-gpu
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME: console-plugin-nvidia-gpu
+LAST DEPLOYED: Tue Aug 23 15:37:35 2022
+NAMESPACE: nvidia-gpu-operator
+STATUS: deployed
+REVISION: 1
+NOTES:
+View the Console Plugin NVIDIA GPU deployed resources by running the following command:
+
+$ oc -n {{ .Release.Namespace }} get all -l app.kubernetes.io/name=console-plugin-nvidia-gpu
+
+Enable the plugin by running the following command:
+
+# Check if a plugins field is specified
+$ oc get consoles.operator.openshift.io cluster --output=jsonpath="{.spec.plugins}"
+
+# if not, then run the following command to enable the plugin
+$ oc patch consoles.operator.openshift.io cluster --patch '{ "spec": { "plugins": ["console-plugin-nvidia-gpu"] } }' --type=merge
+
+# if yes, then run the following command to enable the plugin
+$ oc patch consoles.operator.openshift.io cluster --patch '[{"op": "add", "path": "/spec/plugins/-", "value": "console-plugin-nvidia-gpu" }]' --type=json
+
+# add the required DCGM Exporter metrics ConfigMap to the existing NVIDIA operator ClusterPolicy CR:
+oc patch clusterpolicies.nvidia.com gpu-cluster-policy --patch '{ "spec": { "dcgmExporter": { "config": { "name": "console-plugin-nvidia-gpu" } } } }' --type=merge
+
+----
++
+The dashboard relies mostly on Prometheus metrics exposed by the NVIDIA DCGM Exporter, but the default exposed metrics are not enough for the dashboard to render the required gauges. Thus, the DGCM exporter should be configured to expose a custom set of metrics, as shown here.
```

Comment:
Recommend "therefore" vs "thus". Recommend "is" vs should be"

Reply:
corrected.